src/test/javascript/spec/component/programming-assessment/programming-assessment-repo-export-dialog.component.spec.ts

```diff
@@ -139,4 +139,30 @@ describe('ProgrammingAssessmentRepoExportDialogComponent', () => {
         expect(comp.exportInProgress).toBeFalse();
         expect(exportReposStub).toHaveBeenCalledTimes(2);
     }));
+
+    describe('Export a repo with excludePracticeSubmissions as true', () => {
+        const httpResponse = createBlobHttpResponse();
+
+        beforeEach(() => {
+            fixture.detectChanges();
+            comp.repositoryExportOptions.excludePracticeSubmissions = true;
+        });
+
+        it('should call exportReposByParticipations with correct options', fakeAsync(() => {
+            comp.participationIdList = participationIdList;
+            const exportReposStub = jest.spyOn(repoExportService, 'exportReposByParticipations').mockReturnValue(of(httpResponse));
```

Comment:
I think you might be able to more that outside of `it` and inside your inner `describe` function call as well.

Reply:
Yeah I know, but if you do it it gets less readable while not being any shorter. It seems nicer like this for now.

Reply:
Ok!